src/Sulu/Bundle/AdminBundle/Resources/js/containers/Login/LoginForm.js

```diff
@@ -60,7 +60,10 @@ class LoginForm extends React.Component<Props> {
 
         const {onSubmit} = this.props;
 
-        onSubmit(this.user, this.password);
+        onSubmit({
+            user: this.user,
+            password: this.password,
```

Comment:
this is the component which I want to overwritein my project and add here additional fields which should be added to the loginRequest